godel_simple_gui/src/states/select_plans_state.cpp

```diff
@@ -33,17 +31,14 @@ void godel_simple_gui::SelectPlansState::onStart(BlendingWidget& gui)
   }
 
   if (plan_names_.empty())
+    Q_EMIT newStateAvailable(new SelectAllSurfaceState());
```

Comment:
If any one part of an if-else statement uses curly braces then the other part must too.  See [here](https://google.github.io/styleguide/cppguide.html#Conditionals)